libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -33,11 +35,34 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.searchForm.get('term').valueChanges
+    .pipe(
+    // debounce input for 500 milliseconds
+    debounceTime(500),
+    // only emit if emission is different from previous emission
+    distinctUntilChanged(),
+    // checking if form is valid
+    //filter(() => this.searchForm.valid),
```

Comment:
Remove comments and commented code from this `ngOnInit()` method

Reply:
Done